react/components/GalleryLayoutItem.tsx

```diff
@@ -32,8 +32,14 @@ const GalleryLayoutItem: React.FC<GalleryLayoutItemProps> = ({
     }
   }, [searchQuery])
 
+  const map = useMemo(() => {
+    if (searchQuery?.variables) {
+      return searchQuery.variables.map
+    }
+  }, [searchQuery])
```

Comment:
I think it's better to use the variable directly so the memo runs fewer times 
```suggestion
  const map = useMemo(
    () => searchQuery?.variables?.map,
    [searchQuery?.variables?.map]
  )
```

Reply:
Nice, thank you! 😊